### ISSUE TITLE
Not working for certain plugins... Possible solution?

### ISSUE THREAD

**Issue by WildeWebMarketing** [YahnisElsts/wp-update-server#85]
I'm not sure if you're still supporting this, but I've been using your code for quite some time now.

For most plugins, new updates notices and updating works perfectly.

But...

What I've noticed is that no matter what I do, certain plugins are not showing updates. According to Query Monitor the API request is successful and manually plugging the request into the browser URL returns the correct information. But the plugins never show updates available.

Just looking for a little guidance to possibly narrow down and fix this problem. Thank you kindly.

**Comment by YahnisElsts** [comment]
Yes, this project is still supported - at least in the sense that I use it for a few of my own plugins and if it suddenly breaks for some reason I would probably notice and fix it.

Here are a few of things that I would recommend:

- Verify that you are giving the update checker the correct plugin file name (that is, the full path to the file that contains the plugin header). If you pass in an incorrect file name, or a valid path to some other file, the update checker might still run and even check for updates, but those updates will never show up.
- Are these specific plugins installed in `mu-plugins`? WordPress doesn't support automatic updates for such plugins, but you could make update notifications show up with some additional configuration.
- Install [Debug Bar](https://wordpress.org/plugins/debug-bar/) and look at the "PUC (plugin-slug)" panel. Does all of the information shown there look correct? Try clicking "Request Info" and "Check Now" - does it show the expected results?
